man/git-ftp.1.md

```diff
@@ -189,6 +189,26 @@ But, there is not just FTP. Supported protocols are:
 `ftpes://...`
 :	FTP over explicit SSL (FTPES) protocol
 
+# EXAMPLES
+
+Upload your files to an FTP server the first time:
+
+	$ git ftp init -u "$USER" -P "ftp://example.com/public_html"
+
+It will authenticate with the username stored in `$USER` and ask for the
+password. Using SFTP is a much better option. But be aware of the different
+handling of relative and absolute paths. If the directory `public_html` is in
+the home directory on the server, then upload like this:
+
+	$ git ftp init -u "$USER" --key "$HOME/.ssh/id_rsa" "sftp://example.com/~/public_html"
+
+Otherwise it will use an absolute path, for example:
+
+	$ git ftp init -u "$USER" --key "$HOME/.ssh/id_rsa" "sftp://example.com/var/www"
+
+It is quite common to deactivate server certificate checking with the
+`--insecure` option.
```

Comment:
Thank you. This made me research a bit more about the connection between the public key and the insecure flag. I found in the current manual of curl that all curl versions before 7.39.0 need the public key. Git-ftp guesses the path of the public key so that you have to point it only to the private key though.

More confusing was the insecure flag. On my system, it is automatically added (against the intention of the code). On your system it seems that it is not added. Can you check that for me?

Anyway, Curl is supposed to need the insecure flag only if the server is not in the known_hosts file. Unfortunately, there are some incompatibilities. My version of curl checks for the MD5 fingerprint of the server. But my known_hosts file contains a more secure version of the server's fingerprint. So they don't match and I need the insecure flag. Does the same apply to you with OSX?

What do you think about this version?
```
        $ git ftp init -u "$USER" --key "$HOME/.ssh/id_rsa" --insecure "sftp://example.com/var/www"

The `--insecure` option is needed in most cases. Curl often fails to verify the server's fingerprint with the system's known_hosts file.
```

I'm also wondering if I should change the code to use the `--insecure` flag by default on all systems. That would make it easier for a lot of people. I could also make sure that insecure is not used by default on all systems. But that could cause some currently working scripts to fail. :-(